content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Getting started with Integrations
+kind: documentation
+aliases:
+  - /getting-started-integrations/
+  - /guides/getting-started-integrations/
+  - /guides/integrations/
+further_reading:
+- link: "https://learn.datadoghq.com/course/view.php?id=2"
+  tag: "Learning Center"
+  text: "Introduction to Datadog"
+- link: "/integrations/"
+  tag: "Integrations"
+  text: "Our full list of integrations"
+- link: "/graphing/"
+  tag: "Graphing"
+  text: "Now that you have your integrations set up, build graphs to visulize your data"
+- link: "/monitors/"
+  tag: "Alerts"
+  text: "Set up alerts on your data"
+---
+
+An integration at the highest level is when you assemble a unified system from units that are usually considered separately. At Datadog, you can use integrations to bring together all of the units of your infrastructure and gain insight into the unified system as a whole - you can see pieces individually and also how individual pieces are impacting the whole. Datadogs integrations use a Python class called a check to define the typical metrics most people would want to monitor for that system.
+
+* Over 250 integrations are [officially listed][1], and supported in our [integrations core][2] (and we're always adding more).
+* Even more integrations have been built by our open source community, in [integrations extras][3].
+* You can build your own integration using [Datadog API][4].
+* The Agent is [open source][5] and you can instrument your own if you'd like.
+* All incoming data is treated the same throughout Datadog.
+
+Whether you're beginning developing a new product today or you've been developing for decades, it's never too early or too late to start monitoring with Datadog.
```

Comment:
```suggestion
Whether you're beginning development on a new project today, or you've been developing for decades, it's never too early or too late to start monitoring.
```

changing from "beginning developing" to "beginning development on" because i don't like gerund-gerund it's just weird-sounding

changed product to project, since not everyone uses DD for products; some of them are just using it for hobbies/funsies

removed "with datadog"—with that at the end, the whole sentence comes off as a bit marketingy, but it is true that it's never too early or too late to start monitoring! also having a transitional sentence is nice.

Reply:
Cool. I am trying to answer the question here of "when should I start using this?" I think the answer to that is as early as possible is better, but that being said, you can really start at any time. I think it's important information, but I am not sure how to communicate that without being marketingy